pymc3/smc/smc.py

```diff
@@ -198,47 +198,43 @@ def update_proposal(self):
 
     def tune(self):
         """
-        Tune scaling and n_steps based on the acceptance rate.
+        Tune n_steps based on the acceptance rate.
         """
-        ave_scaling = np.exp(np.log(self.scalings.mean()) + (self.acc_per_chain.mean() - 0.234))
-        self.scalings = 0.5 * (
-            ave_scaling + np.exp(np.log(self.scalings) + (self.acc_per_chain - 0.234))
-        )
-
         if self.tune_steps:
             acc_rate = max(1.0 / self.proposed, self.acc_rate)
             self.n_steps = min(
-                self.max_steps, max(2, int(np.log(1 - self.p_acc_rate) / np.log(1 - acc_rate))),
+                self.max_steps,
+                max(2, int(np.log(1 - self.p_acc_rate) / np.log(1 - acc_rate))),
             )
 
         self.proposed = self.draws * self.n_steps
 
     def mutate(self):
         ac_ = np.empty((self.n_steps, self.draws))
 
-        proposals = (
-            np.random.multivariate_normal(
-                np.zeros(self.dimension), self.cov, size=(self.n_steps, self.draws)
-            )
-            * self.scalings[:, None]
-        )
         log_R = np.log(np.random.rand(self.n_steps, self.draws))
 
+        dist = multivariate_normal(self.posterior.mean(axis=0), self.cov)
+
         for n_step in range(self.n_steps):
-            proposal = floatX(self.posterior + proposals[n_step])
+            proposal = floatX(dist.rvs(size=self.draws))
+            proposal = proposal.reshape(len(proposal), -1)
+            forward = dist.logpdf(proposal)
+            backward = multivariate_normal(proposal.mean(axis=0), self.cov).logpdf(self.posterior)
```

Comment:
I want to understand a bit better what the changes are doing here, so my summary is that:

1, it removes the tuning related to `self.scalings`, which originally is to tune a scaling of the proposal so that _each chain_ approach a target acceptance rate `0.234`;

2, the new transition kernel use global information from the current posterior `mu = self.posterior.mean(axis=0)` to build a MVNormal, and to account for the transition 
![image](https://user-images.githubusercontent.com/12952641/93635475-a1318380-f9f2-11ea-8399-20f061688380.png)
 (notation from [Wikipedia](https://en.wikipedia.org/wiki/Metropolis%E2%80%93Hastings_algorithm)).
For that, the "backward" 
![image](https://user-images.githubusercontent.com/12952641/93635523-bb6b6180-f9f2-11ea-8f36-43a30f185fcc.png)
 use a new MvNormal with `mu = mean(x_prime)`. 
It is independent MH as we have 
![image](https://user-images.githubusercontent.com/12952641/93635619-e5bd1f00-f9f2-11ea-8951-59253e5f331c.png)

Reply:
Right!

Reply:
Cool - could you add some of these as inline comments?

Reply:
Any reason why `dist = multivariate_normal(self.posterior.mean(axis=0), self.cov)` is not in the `for n_step ...` loop?

Reply:
Intuitively I thought about fixing the proposal before doing a productive run. Also because this will make easier to use more complex proposal distributions in the future, as you compute the mean(s) and covariance(s) one per stage.